test/e2e/framework/util.go

```diff
@@ -478,7 +478,7 @@ func isIPv6Endpoint(e *v1.Endpoints) bool {
 				continue
 			}
 			// Endpoints are single family, so it is enough to check only one address
```

Comment:
update this comment?
/lgtm cancel

Reply:
why, it is still the same, we only check one address to know the family

Reply:
the bug was despite iterating over the Addresses slice, we always picked the one on the 0 position ... but we still conside only one address, the first that we found. The name maybe misleading with the API field, but address = IP address